src/PresentationalComponents/PlainTextWithLink.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Text, TextVariants } from '@patternfly/react-core';
+
+export const PlainTextWithLink = ({ label, linkText, linkUrl }) => {
+    return (
+        <Text component={ TextVariants.p }>
```

Comment:
The way this is structured is probably not flexible enough. What, if there are 3 words and the link, should be middle one?

Can we come up with some template string-like syntax? For instance if we check the react-intl format message API we could endup with something like this:

```jsx
// input
Foo <link>bar</link> bazz
// output
<Text {...someProps}>Foo <Text component="a" {...moreProps}>bar</Text> bazz</Text>
```
It should be easy enough to replace the rich text tag with react component using string operations

Reply:
As I proposed some time ago, we could go fully markdown way as we need store the schema in the API. But maybe it's too complex to just show a text with a link.

Reply:
I think this is a common enough component that will require fairly powerful customization later on. I ain't a huge fan of adding MD dependencies just for a simple link component but I can see the appeal. We could "steal" the MD way of defining the link though. One way or another I think we need a bit more flexible definition that this one.

Reply:
@Hyperkid123 - I am not sure how can we replace "<link>" with "<Text component...>" mind dropping some refs?
Thanks.

Reply:
@boaz0 you could use regexp for instance. You can find the non-link string like this: 
```js
"Foo <link>bar</link> bazz".split(/<link>.*<\/link>/) // output => ["Foo ", " bazz"]
```

And then extract the text inside the link tag and create a react link component

```js
const x = "Foo <link>bar</link> bazz"
x.slice(x.search(/<link>/) + 6, x.search(/<\/link>/)) // output => bar
```
Now you have all 3 fragments and you can compose the component.

Reply:
IMHO, sounds a little bit odd and not secure (DDoS attack, idk). But if that is what you want, then I guess we can do that.

Reply:
What do you mean by DDos atack?

Reply:
Let me rephrase myself. I do **not** think this is a bad idea. I am just concerned that it is done in run-time versus in build time. If you are OK with this, I don't mind. I also was hoping there was a function/method/component that we can use in React Intl that can do it for us but I don't care doing it.

Reply:
Right. The runtime risk is minimal IMO. This would do basically the same thing as your initial implements but is slightly more flexible. You can definitely use react intl if you want. It has rich text API with exactly the same syntax. And obviously their implementation is much more sophisticated.